tools/tests/README.md

```diff
@@ -0,0 +1,98 @@
+<!--Copyright (c) Coherent Labs AD. All rights reserved. -->
+# Testing the components
+
+The components are tested using [Jasmine](https://jasmine.github.io/) for assertions

```

Comment:
Changed.

Reply:
Also added a couple of lines explaining how to inspect the page in Chrome and Gameface(as per suggested in the PR - good point). I considered adding the info about Gameface somewhere in the private repo, but I think that having everything in one place is better. Customers who use Gameface will most probably read this instead of some random file located in the private repo. We already mention Gameface in the public docs, so it won't hurt.